app/Models/PasswordResetAuthentication.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class PasswordResetAuthentication extends Model
+{
+    use HasFactory;
+
+    protected $fillable = [
+        'token',
+        'user_id',
+        'status',
+        'expired_at'
+    ];
+
+    public function user()
+    {
+        return $this->belongsTo(User::class, 'user_id', 'id');
```

Comment:
```suggestion
        return $this->belongsTo(User::class);
```
これでも動くかとおもいます。
他にも同じような箇所あるので、確認してください。

Reply:
モデル内のメソッドの引数の削除を行いました。
ご確認よろしくお願い致します。
app/Models/Task.php
app/Models/User.php